fibbingnode/southbound/lsdb.py

```diff
@@ -466,7 +505,30 @@ def process_lsa(self):
                                  for part in lsa_info.split(SEP_GROUP) if part]
                     hdr = LSAHeader.parse(lsa_parts.pop(0))
                     lsa = LSA.parse(hdr, lsa_parts)
+                    new_seqnum = hdr.lsa_seqnum
+
                     log.debug('Parsed %s: %s', action, lsa)
+
+                    # For now, we just log lsa seq_num and ignore an LSA when
+                    # it has an older seq_num than the one in the database
+                    if hdr.lsa_type == '5':
```

Comment:
Sure :+1: 
Do you like the way the sequence number is currently saved? Do you think it is useful to have 3 different sequence number dictionaries (one for router, network and external lsa), or should we just have a single one?